Fuzzing/santactl/src/main.mm

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <cstdint>
+#include <vector>
+
+#include <SNTCommandSyncRuleDownload.h>
+#include <SNTCommandSyncState.h>
+#include <SNTCommandSyncConstants.h>
+#include <SNTRule.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const std::uint8_t *data, std::size_t size) {
+  NSData* buffer = [NSData dataWithBytes:static_cast<const void *>(data) length:size];
```

Comment:
To be consistent with the rest of the source, the * should be with the variable name:

`NSData *buffer`